test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -102,6 +142,27 @@ class test_ucp_rma : public test_ucp_memheap {
                            (uintptr_t)target_ptr, rkey, &param);
     }
 
+    ucs_status_ptr_t do_put_iov(size_t size, void *target_ptr, ucp_rkey_h rkey,
+                                void *expected_data, ucp_request_param_t *param,
+                                ucp_dt_iov_t *iov, size_t iov_count, void *arg) {
+        
+        const size_t iov_buffer_length = size / iov_count;
+        ucs_memory_type_t *mem_types   = reinterpret_cast<ucs_memory_type_t*>(
+                arg);
+        mem_buffer::pattern_fill(expected_data, size, ucs::rand(),
```

Comment:
The `pattern_fill` issue should be handled in a separate PR (changes in several files, unrelated to the iov zcopy change).